File: ios/ios-initials/HW/HW/ViewController.m
//
//  ViewController.m
//  HW
//
//  Created by Zappts - Apple 1 on 12/16/16.
//  Copyright © 2016 Zappts Soluções. All rights reserved.
//

#import "ViewController.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view, typically from a nib.
}


- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}


- (IBAction)showView:(id)sender {
    [UIView beginAnimations:nil context:nil];
    self.workingView.alpha = 1;
    [UIView commitAnimations];
}

- (IBAction)hideView:(id)sender {
    [UIView beginAnimations:nil context:nil];
    self.workingView.alpha = 0;
    [UIView commitAnimations];
}

@end

File: ios/ios-initials/Silly-Song/Silly Song/ViewController.swift
//
//  ViewController.swift
//  Silly Song
//
//  Created by Zappts - Apple 1 on 6/22/17.
//
//

import UIKit

class ViewController: UIViewController {
    
    
    
    @IBOutlet weak var nameField: UITextField!
    @IBOutlet weak var lyricsView: UITextView!
    
    
    override func viewDidLoad() {
        // Do any additional setup after loading the view, typically from a nib.
        super.viewDidLoad()
        nameField.delegate = self
        
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }
    
    @IBAction func reset(sender: AnyObject) {
    }
    
    @IBAction func displayLyrics(sender: AnyObject) {
    }

}

// MARK: - UITextFieldDelegate
extension ViewController: UITextFieldDelegate {
    func textFieldShouldReturn(_ textField: UITextField) -> Bool {
        textField.resignFirstResponder()
        return false
    }
}

